website/docs/r/rde_interface_behavior.html.markdown

```diff
@@ -47,12 +47,22 @@ resource "vcd_rde_interface_behavior" "my_behavior2" {
 resource "vcd_rde_interface_behavior" "my_behavior3" {
   rde_interface_id = vcd_rde_interface.my_interface.id
   name             = "MyBehavior3"
-  execution = {
+
```

Comment:
Why do we remove previous `execution` way from example? Do we prefer `execution_json` exclusively?

Reply:
It got me a few mins to understand what was going on here, as I didn't remove the `execution` example, but added a new one with `execution_json`. I think it's just GitHub diff view messing around, if you open the rendered file, you can see both are there now.

Reply:
Oh, indeed! Thanks!